upgrades/gov-props/v1.2.0.md

```diff
@@ -1,28 +1,33 @@
-# v1.2.0 - Bouncy Bulldog
-
+# v1.2.0
 ## Summary
 This upgrade allows the Jackal Network to start fulfilling its purpose as the Decentralized Storage Platform of the 
 Cosmos. This upgrade introduces the Storage module, which allows users to pay for storage and create storage deals with 
 Storage Providers. It also brings the addition of the Filetree module which creates a privacy layer and bucket file 
 structure akin to S3 and Google Drive. Finally, it also brings the addition of the Oracle module which allows other 
 modules to query a data stream being supplied by a centralized entity. In this case, Jackal Labs will be using this 
-oracle to supply pricing information while the AMM module is being finalized. 
+oracle to supply pricing information while the AMM module is being finalized.
+
+The upgrade will take place at block height 2080380, estimated to be on March 23rd. This upgrade marks a crucial 
+milestone for the Jackal Protocol and brings it one step closer to achieving the vision of providing a decentralized and 
+efficient storage platform for the Cosmos ecosystem and beyond.
 ## Change Log
 * Re-added Storage Module
 * Re-added Filetree Module
+* Re-added Notification Module
 * Added Oracle Module
 * Fixed RNS marketplace bugs
 * Increased Code Test Coverage to ~50%
 * Fixed Go Package & Proto Layout
 * Removed Ignite CLI as a dependency & fixed codegen bugs associated with it
 * Improved Safety of Storage/Filetree modules from v1.1.1
-* Updated Cosmwasm Version from v0.29.1 -> v0.29.2
-* Updated CosmosSDK Version from v0.45.10 -> v0.45.14
+* Updated Cosmwasm Version from v0.29.1 -> v0.30.0 w/ Patch
```

Comment:
Master branch is only at Cosmwasm v0.29.2-patch.1. Do we plan on making this update in the future? @TheMarstonConnell

Reply:
Oh I thought it was at 30, yeah I'll roll it back